From 80523598da24818579b2321fd52fa2f3a2cab8a9 Mon Sep 17 00:00:00 2001
From: Michal Simek <michal.simek@xilinx.com>
Date: Thu, 22 Nov 2012 14:11:00 +0100
Subject: [PATCH] usb: host: Remove OF dependency in USB_EHCI_XUSBPS

USB_EHCI_XUSBPS depends on ARCH_ZYNQ which has always
OF on. That's why another OF dependency was removed.

Signed-off-by: Michal Simek <michal.simek@xilinx.com>
---
 drivers/usb/host/Kconfig |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/usb/host/Kconfig b/drivers/usb/host/Kconfig
index 7bd3da8..1ad40ff 100644
--- a/drivers/usb/host/Kconfig
+++ b/drivers/usb/host/Kconfig
@@ -138,7 +138,7 @@ config USB_EHCI_XUSBPS
 	bool "Support for Xilinx PS EHCI USB controller"
 	depends on USB_EHCI_HCD && ARCH_ZYNQ
 	select USB_EHCI_ROOT_HUB_TT
-	select USB_XUSBPS_DR_OF if OF
+	select USB_XUSBPS_DR_OF
 	---help---
 		Xilinx PS USB host controller core is EHCI compilant and has
 		transaction translator built-in.
-- 
1.7.0.4

